contracts/BtcLightClient.sol

```diff
@@ -41,10 +39,12 @@ contract BtcLightClient is ILightClient, System, IParamSubscriber{
   uint256 public constant ROUND_SIZE=100;
   uint256 public constant MAXIMUM_WEIGHT=20;
   uint256 public constant CONFIRM_BLOCK = 6;
-  uint256 public constant INIT_ROUND_INTERVAL = 86400;
   uint256 public constant POWER_ROUND_GAP = 7;
   uint256 public constant INIT_STORE_BLOCK_GAS_PRICE = 35e9;
 
+  uint32 public immutable INIT_CHAIN_HEIGHT;
```

Comment:
I prefer upper-casing immutables since, past the constructor, they are effectively constants